CHANGELOG.rst

```diff
@@ -22,9 +22,9 @@ Each individual change should have a link to the pull request after the descript
 Added
```

Comment:
please could you update the changelog version number and release date (to today)

Reply:
updated date here https://github.com/lvgig/tubular/pull/174/commits/5a5815546e34348b32cb7d605294dbd9ee979c78

and version here
https://github.com/lvgig/tubular/pull/174/commits/b10649399f759695c69f7b9cf602ef825c486c67